README.md

```diff
@@ -1,28 +1,102 @@
-# inmemory-pagination
+# In Memory Pagination
 
-## Filter & Search :
 
-Filtering is done by column name while search is performed across columns.
-Multiple values can be supplied at the same time.
+### Problem Statement
 
-`&` -> separates column names
-`|` -> separates multiple values
-`*` -> indicates search across all columns
+Struck with Database with no filtering,searching and pagination features?.
+UI is feeling Hard to handle bulk of data?.
+We provide you the best solution to overcome these issues with minimal overhead.
 
-### Examples:
+### Solution
 
-"name=bharath"
-"*=Vega|vegas"
-"multiMedia.[].name=CCCC&*=Vega|vegas&userId=4051"
+As we are developing so much software in today’s world and dealing with tons of data.
+Imagine a situation where you have tons of data to show on your web page.
+A browser will have not able to render all the data at one shot.
+So the best option is to go for InMemory pagination, where a server will return the bounded number of rows to show in the UI.
 
-## TODO : 
 
-1. write logic for filtering
-2. write logic for searching
-3. add enough log statement
-4. extend logic to support nested objects ( filtering, search and sort)
-5. create excception class
-6. Throw exception column name is invalid(if column name is not exist)
-7. Write unit testcases (use cucumber)
-8. document on how to use with example
-9. Finally publish to maven central repo
+### How to use this plugin
```

Comment:
Now you have to include two options on how to use this plugin

Reply:
i have changed it .. please look into it